### PR TITLE
add esa sss as example tutorial

### PR DESCRIPTION
**Pull request by rwegener2**
Parts 1 and 2 of a tutorial have been copied into a new folder to use ESA SSS

one of the images isn't rendering so I'm circling back on that.

### CODE REVIEW

**Comment by rabernat** [comment]
Thanks for working on this Rachel.

But I'm a bit skeptical that we want to be pulling data from Europe for this tutorial. Our binders are all in the US!

**Comment by rwegener2** [comment]
How do you feel about https://coastwatch.noaa.gov/pub/socd/lsa/rads/sla/daily/dt/  @rabernat ?

**Comment by rabernat** [comment]
That should be better I think. Does it work?

**Comment by rwegener2** [comment]
It runs but the output creates a blank map so not sure what that's about

File: docs/introduction_tutorial_ocean_sciences/intro_tutorial_part2.ipynb [hunk 1]
```diff
@@ -0,0 +1,1950 @@
+{
```
Comment (rabernat) [inline docs/introduction_tutorial_ocean_sciences/intro_tutorial_part2.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    sla_zarr['sla'].sel(time='2012-01-02').plot()
</pre><p>add robust=True</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-forge/pangeo-forge-recipes/pull/314/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

**Pull request merged**